examples/slack_alert/run_ask.py

```diff
@@ -0,0 +1,39 @@
+from steps import evaluator, importer, svc_trainer_mlflow
+
+from zenml.alerter.alerter_step import alerter_ask_step
+from zenml.integrations.mlflow.steps import mlflow_model_deployer_step
+from zenml.pipelines import pipeline
+from zenml.steps import step
+
+
+@step
+def test_acc_formatter(test_acc: float) -> str:
+    """Wrap given test accuracy in a nice text message."""
+    return (
+        f"Model training finished. Test accuracy: {test_acc}. "
+        f"Deploy now? [approve | reject]"
+    )
+
+
+@pipeline(enable_cache=False)
+def slack_ask_pipeline(
+    importer, trainer, evaluator, formatter, alerter, deployer
+):
+    """Train and evaluate a model and asks user in Slack whether to deploy."""
+    X_train, X_test, y_train, y_test = importer()
+    model = trainer(X_train=X_train, y_train=y_train)
+    test_acc = evaluator(X_test=X_test, y_test=y_test, model=model)
+    message = formatter(test_acc)
+    approved = alerter(message)
+    deployer(approved, model)
```

Comment:
You are already following the best practices half way by outsourcing some steps, maybe it would be good to fully commit and put all steps and all pipelines in the appropriate folders?